testdata/p4tc_samples_outputs/simple_extern_example.template

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/simple_extern_example numtables 1
+
+$TC p4template create action/simple_extern_example/ingress/send_nh actid 1 \
+	param port_id type dev \
+	param dmac type macaddr \
+	param smac type macaddr
+$TC p4template update action/simple_extern_example/ingress/send_nh state active
+
+$TC p4template create action/simple_extern_example/ingress/ext_reg actid 2 \
+	param port_id type dev
+$TC p4template update action/simple_extern_example/ingress/ext_reg state active
+
+$TC p4template create action/simple_extern_example/ingress/drop actid 3
+$TC p4template update action/simple_extern_example/ingress/drop state active
+
+$TC p4template create table/simple_extern_example/ingress/nh_table \
+	tblid 1 \
+	type lpm \
+	keysz 32 nummasks 8 permissions 0x3da4 tentries 262144 \
+	table_acts act name simple_extern_example/ingress/ext_reg \
+	act name simple_extern_example/ingress/send_nh \
+	act name simple_extern_example/ingress/drop
+$TC p4template update table/simple_extern_example/ingress/nh_table default_miss_action permissions 0x1024 action simple_extern_example/ingress/drop
+
+$TC p4template create extern/simple_extern_example/root/Register extid 1 numinstances 2 tc_acl 0x19b6
```

Comment:
Minor nit, remember that in case of extern template pipeline name is always root

```suggestion
$TC p4template create extern/root/Register extid 1 numinstances 2 tc_acl 0x19b6
```

Reply:
I saw this comment late, PR is merged, I will fix this in next PR.